UIKitVK/UIKitVK/Controller/NewsViewController.swift

```diff
@@ -60,16 +92,25 @@ final class NewsViewController: UIViewController {
                 item.avatarURL = user.friendAvatar
             }
         }
-        news = response.news
-        newsTableView.reloadData()
+        return response.news
+    }
+
+    @objc private func refreshNewsAction() {
+        fetchNews()
+        refreshControl.endRefreshing()
     }
 }
 
 // MARK: - UITableViewDataSource, UITableViewDelegate
 
-extension NewsViewController: UITableViewDataSource, UITableViewDelegate {
+extension NewsViewController: UITableViewDataSource, UITableViewDelegate, UITableViewDataSourcePrefetching {
```

Comment:
```suggestion
extension NewsViewController: UITableViewDataSource, UITableViewDelegate {
```
 в отдельный extension